snmp/README.md

```diff
@@ -236,46 +127,314 @@ instances:
+| `snmp.tcpHCOutSegs`                    | The total number of segments sent, including those on current connections but excluding those containing only retransmitted octets.                                                                                                                                       |                          |
+| `snmp.tcpRetransSegs`                  | The total number of segments retransmitted; that is, the number of TCP segments transmitted containing one or more previously transmitted octets.                                                                                                                         |                          |
+| `snmp.tcpInErrs`                       | The total number of segments received in error (e.g., bad TCP checksums).                                                                                                                                                                                                 |                          |
+| `snmp.tcpOutRsts`                      | The number of TCP segments sent containing the RST flag.                                                                                                                                                                                                                  |                          |
+| `snmp.udpHCInDatagrams`                | The total number of UDP datagrams delivered to UDP users, for devices that can receive more than 1 million UDP datagrams per second.                                                                                                                                      |                          |
+| `snmp.udpNoPorts`                      | The total number of received UDP datagrams for which there was no application at the destination port.                                                                                                                                                                    |                          |
+| `snmp.udpInErrors`                     | The number of received UDP datagrams that could not be delivered for reasons other than the lack of an application at the destination port.                                                                                                                               |                          |
+| `snmp.udpHCOutDatagrams`               | The total number of UDP datagrams sent from this entity, for devices that can transmit more than 1 million UDP datagrams per second.                                                                                                                                      |                          |
+
+##### F5 BIG-IP profile
+
+The [F5 BIG-IP profile][8] collects the following metrics.
+
+MIBs needed for local reference: F5-BIGIP-SYSTEM-MIB, IF-MIB.
```

Comment:
You should be able to remove that for 7.17+ too.

Reply:
updated and removed the reference